opt_base_2_bytes.go

```diff
@@ -0,0 +1,98 @@
+package configtypes
+
+import (
+	"encoding/json"
+
+	"github.com/alecthomas/units"
+
+	"gopkg.in/launchdarkly/go-sdk-common.v2/ldvalue"
```

Comment:
Oh wow, this is the really old version of go-sdk-common. I think this should probably be updated to the current package, `github.com/launchdarkly/go-sdk-common/v3`

Reply:
I can do that in another PR.